tests/views/test_collection_exercise.py

```diff
@@ -2767,3 +2767,11 @@ def test_add_collection_instrument_duplicate(self, mock_request):
 
         self.assertEqual(response.status_code, 200)
         self.assertIn("There is a problem with this page".encode(), response.data)
+
+    def test_view_ci_versions(self):
```

Comment:
I think you are a little light here, as there are other pages with 0001 on, perhaps have "Choose CIR version for EQ formtype" as well